### PR TITLE
Add more byte array output formats (Python, Kotlin, Java)

### PR DESCRIPTION
**Pull request by jmg-duarte**
This is useful when dealing with shellcode for example or other loading of a binary to a Python byte array which may serve as a payload

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,6 +126,15 @@ a := [3]byte{
...
+```sh
+$ hx -ap -c8 tests/files/tiny.txt
+a = [
+    0x69, 0x6c, 0x0a
+]
+```
+
 ## manual
 
 ```txt
```
Comment (sitkevij) [inline README.md:158]: @jmg-duarte thank you for you contribution. could you also update the manual section `Set source code format output` to include the Python array output?

Comment (jmg-duarte) [reply README.md:158]: Will take care of that, also if you're interested I can add more languages, such as Kotlin and Java

Comment (sitkevij) [reply README.md:158]: great, was just looking at adding Kotlin recently. feel free to add support for Kotlin and Java.

**Comment by jmg-duarte** [comment]
Just added support for Kotlin and Java, I also updated the PR name

**Pull request merged**